quil-rs/src/program/mod.rs

```diff
@@ -407,12 +418,12 @@ DEFCAL I 0:
     DELAY 0 1.0
 DEFFRAME 0 \"rx\":
     HARDWARE-OBJECT: \"hardware\"
-DEFWAVEFORM custom 6.0:
+DEFWAVEFORM custom:
     1,2
 I 0
 ";
-        let a = Program::from_str(input);
-        let b = Program::from_str(input);
+        let a = Program::from_str(input).unwrap();
+        let b = Program::from_str(input).unwrap();
```

Comment:
The parsing error was not caught because we were inadvertently testing that the errors were equal, not the parsed programs.